signer/src/ecdsa.rs

```diff
@@ -109,7 +112,8 @@ impl Keypair {
     pub fn from_seed(seed: Seed) -> Result<Self, Error> {
         let secret = SecretKey::from_slice(&seed).map_err(|_| Error::InvalidSeed)?;
         Ok(Self(secp256k1::Keypair::from_secret_key(
-            SECP256K1, &secret,
+            &Secp256k1::signing_only(),
```

Comment:
yeah ok, this probably because they some lazy_static stuff that doesn't compile for no-std?

then removing global-context makes sense